package.json

```diff
@@ -11,7 +11,7 @@
     "release": "npm login && release-it",
     "type:check": "tsc"
   },
-  "main": "lib/commonjs/Animated",
+  "main": "src/Animated.js",
```

Comment:
BTW, this is from bob config, seems like we don't use it right now. Could we reintroduce it or the way how we build android is preventing us from doing it?